utility-emissions-channel/typescript_app/package.json

```diff
@@ -19,13 +19,14 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@brioux/cactus-plugin-ledger-connector-fabric": "^0.9.8-web-socket-identity-provider",
```

Comment:
@brioux , din't @petermetz  released the fabric connector for your fork?

Reply:
the canary release was out of date after changing the ws-identity server. I'll ask him to issue another until the PR is approved an added to version 0.10.0 of cactus